packages/analytics/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.2.0
+
+-   npm-prerelease: @trezor/analytics 1.1.1-beta.1 (69ebdd96a5)
```

Comment:
kinda weird changelog 🤔

Reply:
Yes, this is new because we did not have `npm-prerelease` before, but I thought it can be useful since it is adding information there about there was a previous beta release of the same.
But yeah, a bit weird, I can remove it an change the script so it does not include it next time. Up to you.